SUPPORT.md

```diff
@@ -22,20 +14,24 @@ This isn't the right place to get support for NGINX specific questions, but the
 
 We have a community [Slack](https://nginxcommunity.slack.com/)!
 
-If you are not a member click [here](https://community.nginx.org/joinslack) to sign up (and let us know if the link does not seem to be working!)
+If you are not a member, click [here](https://community.nginx.org/joinslack) to sign up. (Let us know if the link does not seem to be working at <nginx-oss-community@f5.com>!)
 
 Once you join, check out the `#beginner-questions` and `nginx-users` channels :)
 
 ### Documentation
 
 For a comprehensive list of all NGINX directives, check out <https://nginx.org>.
 
-For a comprehensive list of admin and deployment guides for all NGINX products, check out <https://docs.nginx.com>.
+For a comprehensive list of administration and deployment guides for all NGINX products, check out <https://docs.nginx.com>.
 
 ### Mailing List
 
 Want to get in touch with the NGINX development team directly? Try using the relevant mailing list found at <https://mailman.nginx.org/mailman3/lists/>!
 
 ## Contributing
 
-Please see the [contributing guide](CONTRIBUTING.md) for guidelines on how to best contribute to this project.
+Please see the [contributing guide](https://github.com/nginxinc/nginx-s3-gateway/blob/main/CONTRIBUTING.md) for guidelines on how to best contribute to this project.
```

Comment:
nit: We should stick to relative links. Part of the workflow to contribute is to clone the repo which would make these links point at the wrong place.  GitHub recommends this as well https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files

Reply:
I have been hurt in a few places by using relative links, but I guess those places might be the exception to the rule and not the norm. Given GH recommends them as the standard, I am happy to revert back to relative links :)